src/controller/user.controller.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const users = userService.getAll();
+
+  if (!users) {
+    res.send([]);
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  const updatedUser = userService.updateUser({ id: +id, name });
+
+  if (!updatedUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(updatedUser);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+  }
+
+  const user = userService.createUser(name);
+
+  res.statusCode = 201;
```

Comment:
Instead of setting `res.statusCode = 201;`, you can use `res.status(201).send(user);`. It's a more expressive and common way to set the status code and send a response in Express.js.